README.md

```diff
@@ -4,7 +4,7 @@ Welcome to Dispatch's documentation page.
 Dispatch is a Solana protocol to allow dapps and wallets to communicate with each other.
```

Comment:
Dispatch is a monetizable conversational protocol for Solana. It supports two kind of conversation modes - private and public. Private conversations are supported via the Mailbox object, and public conversations via Postbox. Please see below for details.